pkg/oam/labels.go

```diff
@@ -27,6 +27,11 @@ const (
 	LabelAppComponentRevision = "app.oam.dev/revision"
 	// LabelOAMResourceType whether a CR is workload or trait
 	LabelOAMResourceType = "app.oam.dev/resourceType"
+
+	// WorkloadTypeLabel indicates the type of the workloadDefinition
+	WorkloadTypeLabel = "workload.oam.dev/type"
+	// TraitTypeLabel indicates the type of the traitDefinition
+	TraitTypeLabel = "trait.oam.dev/type"
```

Comment:
A little bit inconsistent: In AppConfig, the field name is `spec.components[x].traits[y].name` but the lable key is `type`.

Reply:
Yes, but it's actually a type, I'm aligning with workload here.